### DISCUSSION TITLE
Unordered list hierarchy

### DISCUSSION THREAD

**Discussion by aborruso** [squidfunk/mkdocs-material#2844] [Q&A] [answered]
Hi,
I have this markdown

```
- first 1
- first 2
  - second 2.1
  - second 2.2
```

Why I have no hierarchy in mkdocs-material output? It seems that I must insert 4 withe spaces and not 2.

Thank you

![image](https://user-images.githubusercontent.com/30607/126061080-34802465-952a-4212-94aa-f52548263f6f.png)

**Comment by squidfunk** [comment, marked as answer]
Yes, lists must be indented by four spaces, as this is what Python Markdown expects.

**Reply by aborruso** [reply]
Thank you
